### PR TITLE
(issue): Page Not Found

### PR DESCRIPTION
**Pull request by dancesWithCycles**
Hi folks,
I found two links not resolving properly.

Cheers!

### CODE REVIEW

**Comment by dancesWithCycles** [comment]
Bye the way, could you not decide to write `for convert` of `for building` and instead wrote both in this [line](https://github.com/derhuerst/berlin-gtfs-rt-server/blob/324fd534f3b1b403fb6d73eda012e84e61ca77a8/readme.md?plain=1#L14)? ;-)

Cheers!

File: readme.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This project uses [`hafas-client`](https://github.com/public-transport/hafas-cli
 
 It also needs access to a [PostgreSQL](https://www.postgresql.org) 14+ server; Pass custom [`PG*` environment variables](https://www.postgresql.org/docs/14/libpq-envars.html) if you run PostgreSQL in an unusual configuration.
 
-It also needs access to a [NATS Streaming](https://docs.nats.io/nats-streaming-concepts/intro) server (just follow its [setup guide](https://docs.nats.io/nats-streaming-server/run)); Set the `NATS_STREAMING_URL` environment variable if you run it in an unusual configuration.
+It also needs access to a [NATS Streaming](insert something not resulting in a Page Not Found) server (just follow its [setup guide](insert something not resulting in a Page Not Found)); Set the `NATS_STREAMING_URL` environment variable if you run it in an unusual configuration.
```
Comment (derhuerst) [inline readme.md]:
```suggestion
It also needs access to a [NATS Streaming](https://nats-io.gitbook.io/legacy-nats-docs/nats-streaming-server-aka-stan) server (just follow its [installation guide](https://nats-io.gitbook.io/legacy-nats-docs/nats-streaming-server-aka-stan/stan-nats-streaming-server/installing)); Set the `NATS_STREAMING_URL` environment variable if you run it in an unusual configuration.
```

File: readme.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ docker run -e BBOX -i -t --rm derhuerst/berlin-gtfs-rt-server
 
 ### via docker-compose
 
-The example [`docker-compose.yml`](docker-compose.yml) starts up a complete set of containers (`berlin-gtfs-rt-server`, Redis, PostGIS/PostgreSQL, [NATS Streaming](https://docs.nats.io/nats-streaming-concepts/intro)) to generate a GTFS-RT feed
+The example [`docker-compose.yml`](docker-compose.yml) starts up a complete set of containers (`berlin-gtfs-rt-server`, Redis, PostGIS/PostgreSQL, [NATS Streaming](page not found)) to generate a GTFS-RT feed
```
Comment (derhuerst) [inline readme.md]:
```suggestion
The example [`docker-compose.yml`](docker-compose.yml) starts up a complete set of containers (`berlin-gtfs-rt-server`, Redis, PostGIS/PostgreSQL, [NATS Streaming](https://nats-io.gitbook.io/legacy-nats-docs/nats-streaming-server-aka-stan)) to generate a GTFS-RT feed
```

**Comment by derhuerst** [comment]
Thanks!

**Pull request merged**